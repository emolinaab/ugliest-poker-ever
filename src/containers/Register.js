import React from 'react';

const Register = () => (
  <section></section>
);

export default Register;
