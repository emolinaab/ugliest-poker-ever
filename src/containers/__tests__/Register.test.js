import React from 'react';
import { shallow } from 'enzyme';
import Register from '../Register';

describe('Register Component', () => {
  it('should render correctly', () => {
    const component = shallow(<Register />);
  
    expect(component).toMatchSnapshot();
  });
});
