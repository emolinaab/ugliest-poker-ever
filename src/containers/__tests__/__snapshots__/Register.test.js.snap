// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Register Component should render correctly 1`] = `<section />`;
